src/fulfillment/fulfillment.model.ts

```diff
@@ -24,10 +24,15 @@ const fulfillmentSchema = new Schema(
             type: Date,
             default: () => Date.now()
         },
+        fulfilledStatus: {
+            type: Boolean,
+            default: false,
+            require: true
+        }
```

Comment:
i dont think we need this field on the fulfillment. the idea of a fulfillment being created sort of implies the item is fulfilled.